README.md

```diff
@@ -45,10 +45,11 @@ Publisher<? extends Connection> connectionPublisher = connectionFactory.create()
 ConnectionFactory connectionFactory = ConnectionFactories.get(ConnectionFactoryOptions.builder()
    .option(DRIVER, "postgresql")
    .option(HOST, "...")
-   .option(PORT, "...")  // optional, defaults to 5432
+   .option(PORT, 5432)  // optional, defaults to 5432
    .option(USER, "...")
    .option(PASSWORD, "...")
    .option(DATABASE, "...")  // optional
+   .option(OPTIONS, Map.of("lock_timeout", "30s", "statement_timeout", "5m")) // optional
```

Comment:
Let's not use Java 9+ syntax here as this sample might leave readers with the assumption that they require a newer Java version to run this code. All R2DBC components use Java 8 as baseline.

Reply:
Yeah, but creating a map in Java 8 is sooo ugly and bloats the code, so I chose Java 9 style. I've changed it to J8.